packages/react-components/react-button/src/components/common/InProgress.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { ButtonState } from '../Button/Button.types';
+import { Spinner } from '../../../../react-spinner';
```

Comment:
```suggestion
import { Spinner } from '@fluentui/react-spinner';
```

Also, `@fluentui/react-spinner` needs to be declared in the `package.json` of the `react-button` package.